### ISSUE TITLE
Git dirty segment

### ISSUE THREAD

**Issue by silenc3r** [brujoand/sbp#83]
Most of the time you don't need your status line to show how many lines in git repo have been changed. Simple indicator if something has been changed or not is good enough.

It would be nice if sbp could display git info like https://github.com/rafaelrinaldi/pure, i.e. when there are no modifications just display `master` when there are some modifications display `master*`.

**Comment by brujoand** [comment]
Yeah this should be pretty straight forward to add through a flag `SEGMENTS_GIT_DIRTY_DETAILED=0` which could just clob the different dirty states into ' ' or '*'.
